docs/deploying.md

```diff
@@ -369,6 +369,150 @@ support.
 		* Closing connection 0
 		curl: (60) SSL certificate problem: self signed certificate
 		More details here: http://curl.haxx.se/docs/sslcerts.html
+
+## Deploy a v2 registry with Nginx and basic authentication
+
+This example shows you how to securely deploy a v2 registry using basic authentication.
+
+### Install Docker Compose
+
+1. Open a new terminal on the host with your `distribution` directory.
+
+2. Get the `docker-compose` binary.
+
+		$ sudo wget https://github.com/docker/compose/releases/download/1.2.0/docker-compose-`uname  -s`-`uname -m` -O /usr/local/bin/docker-compose
+
+	This command installs the binary in the `/usr/local/bin` directory. 
+	
+3. Add executable permissions to the binary.
+
+		$  sudo chmod +x /usr/local/bin/docker-compose
+		
+
+### Do some housekeeping
+
+1. Remove any previous images.
+
+		$ docker rmi -f $(docker images -q -a )
+		
+	 This step is a house keeping step. It prevents you from mistakenly picking up
+	 an old image as you work through this example.
```

Comment:
I'll add a warning note or change the command. I do, at this point, want them to clean out specific images.  BTW, the next example uses this technique and I didn't get any feedback that it caused users problems.